### DISCUSSION TITLE
Docker image's root passwd

### DISCUSSION THREAD

**Discussion by PengXuanyao** [gem5/gem5#1884] [Q&A] [answered]
Hi, I'm trying to install some tools(like bear),  on the gem5 docker image [ghcr.io/gem5/ubuntu-24.04_all-dependencies:v24-0](https://ghcr.io/gem5/ubuntu-24.04_all-dependencies:v24-0)
But it needs the `sudo apt install xxx`. Could anybody tell me the root password? 
I couldn't find it in the document. 
Thanks a lot!

**Comment by Harshil2107** [comment, marked as answer]
Hi,  

By default, the Docker image runs as **root**, so you don't need `sudo`. You can directly install packages using:  

```bash
apt install <package-name>
```  

If you want an interactive session with automatic cleanup, you can use:  

```bash
docker run -it --rm ghcr.io/gem5/ubuntu-24.04_all-dependencies:latest
```

**Reply by PengXuanyao** [reply]
Hi,

Thank you! And I have already changed dockerfile to install tools in advance, this also works.
